messages.js

```diff
@@ -129,7 +129,7 @@ exports.embedEventAsImage = async (event, locale) => {
     embed: {
       color: good ? GREEN : RED,
       title,
-      url: KILL_URL.replace("{kill}", event.EventId),
+      url: KILL_URL.replace("{lang}", l.__("KILL_URL_LANG")).replace("{kill}", event.EventId),
```

Comment:
Thanks for your contribution!

You don't need to set a string in every locale to get the current locale. You can use l.getLocale() instead.
Please remove the entries from the locales and use the method mentioned above.